ARCs/arc-0033.md

```diff
@@ -0,0 +1,60 @@
+---
+arc: 33
+title: Becoming an xGov
+description: Explanation on how to become Expert Governors.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+xGov status gives a governor the right to vote on proposals raised by the community, to ratify the proposals as valid for final community vote, where said proposals pertain to spending a previously specified amount of Algo in a given epoch on particular initiatives.
+ 
+The purpose of this ARC is to define the standard by which xGov status is achieved.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+### What is an xGov
+Expert Governors (xGov) are decentralized decision makers who demonstrate enduring commitment, are knowledgeable of the realities of the Algorand community, and whose interests are aligned with the good of blockchain.
+ 
+xGov **SHOULD** understand the community's needs and align their interests with those of the Algorand ecosystem.
+xGov **MAY** propose measures for a vote.
+xGov **MAY** participate in the designation of proposals.
+ 
+### Requirement to become an xGov
+The Foundation suggests that to become an xGov an Algo holder **MUST**:
+- Participate in another reward-bearing AF program
```

Comment:
Is there any chance we can count having "online stake" as a "reward bearing program"? I understand it goes against TVL expansion but it seems like online stake is gradually dropping. (Number of nodes may be growing though).

Reply:
Consensus  participation should be mandatory for xGovs.

Reply:
> I second this. Consensus particupation is also getting less and less governance rewards since those are only going to DeFi. Giving node runners xGov status might at least incentivize them a little.

I don't think it is necessary to run a node to be an xGov, though I'm not against the idea if there are a number of options available to easily onboard this process. That being said, we don't need this to be a requirement to incentivize node runners. The xGov process can be used to propose funding, and I imagine it very much will be almost immediately if something to that effect hasn't been passed before this is implemented.

Reply:
I think Silvio said something like "the hardest part of blockchain is people". Please don't take the quote seriously, it's from a distant memory.

Anyways, I see why incentives can be a double-edged sword. Currently, the only node running incentive is to secure the system that we are using. The moment we change this creative people will come up with creative ways to profit.

The most common example is the formation of staking pools on other chains with monetary incentives. These pools help bring the online stake up but introduce a new form of centralization.

I am not against node runner incentives but I don't think the consequences can be predicted.